src/ekka.app.src

```diff
@@ -12,7 +12,8 @@
    [kernel,
     stdlib,
     mnesia,
-    inets
+    inets%% ,
+    %% gen_rpc
```

Comment:
why commented out ?

Reply:
I need to figure out how to configure it (e.g. map ports) in the existing common test suites.